docs/oathkeeper/index.md

```diff
@@ -100,10 +103,10 @@ request to ORY Oathkeeper.
 
 ### Access Control Decision API
 
-The Access Control Decision API is exposed at `/decision` of the ORY Oathkeeper
-API port. It follows best-practices and works with most (if not all) modern API
-Gateways and Reverse Proxies. The endpoint matches every sub-path and HTTP
-Method:
+The ORY Oathkeeper Access Control Decision API follows best-practices and works with most (if not all) 
```

Comment:
Is it clear enough that `/decision` is exposed at the API port, not the reverse proxy port?

Reply:
Added wording to make that clear.